wpilibj/src/main/java/edu/wpi/first/wpilibj/CameraServer.java

```diff
@@ -335,7 +335,7 @@ private CameraServer() {
             table.getEntry("mode").setDefaultString(videoModeToString(mode));
             table.getEntry("modes").setStringArray(getSourceModeValues(event.sourceHandle));
           } catch (VideoException ignored) {
-            // Do nothing. Let the other event handlers update this if there is an error.
+            // Do nothing. Let the other event handlers calculate this if there is an error.
```

Comment:
I don't think you wanted to touch this.

Reply:
Oops.  Intellij's refactoring tool gets overeager sometimes.